cockatrice/src/carddatabasemodel.cpp

```diff
@@ -181,9 +181,90 @@ bool CardDatabaseDisplayModel::lessThan(const QModelIndex &left, const QModelInd
         if (isRightType && (!isLeftType || rightString.size() == cardName.size()))
             return false;
     }
+    else if (right.column() == CardDatabaseModel::PTColumn && left.column() == CardDatabaseModel::PTColumn) {
+        QStringList leftList = leftString.split("/");
+        QStringList rightList = rightString.split("/");
+
+        if (leftList.size() == 2 && rightList.size() == 2) {
+
+            //cool, have both P/T in list now
+            bool equal;
+            bool lessThanNum = lessThanNumerically(leftList.at(0), rightList.at(0), &equal);
+            if (!equal) {
+                return lessThanNum;
+            }
+            else {
+                //power equal, check toughness
+                return lessThanNumerically(leftList.at(1), rightList.at(1), &equal);
+            }
+        }
+    }
     return QString::localeAwareCompare(leftString, rightString) < 0;
 }
 
+bool CardDatabaseDisplayModel::lessThanNumerically(const QString &left, const QString&right, bool *equal) const {
```

Comment:
I don't think we should use pointer to bool as an "out" parameter. I believe it's idiomatic to return a value in `{-1, 0, 1}` for LT, EQ, GT respectively.

In general, prefer using a pure functional approach.

Reply:
ah ok. That would have been the way I did it in Java, wasn't quite sure what was 'normal' way here. I'll fix that up.